services/QuillLMS/bin/dev/airbyte

```diff
@@ -5,7 +5,7 @@ INSTANCES = {
   1 => {
     name: 'airbyte2',
     port: 8001,
-    tables: "concept_results",
+    tables: "concept_results (prefix: special)",
```

Comment:
Why is this one labeled 'prefix: special' but elsewhere the `special` namespace is being replaced with `lms`?

Reply:
Because we want to keep the legacy instance alive as a fallback, for an unspecified but longish period of time. More discussion here: https://quill.slack.com/archives/C051YCC921L/p1715960113871199